ymcatwincities/openy
# Updated the event listing view page filter.

https://github.com/ymcatwincities/openy/issues/2438

Testing steps: 

- Log in as admin
- Visit an existing page, like /locations/south-ymca
- Edit the page and add both the Upcoming Events and Events Post Listing Paragraphs
-![image](https://user-images.githubusercontent.com/6428296/117697482-22d35d00-b1e0-11eb-8f3c-da411de32287.png)
- Observe events with dates in the past DO NOT appear
- ![image](https://user-images.githubusercontent.com/6428296/117697529-2e268880-b1e0-11eb-88a7-ea2ff1517419.png)
- Add a new event with an end date in the future, observe it DOES appear in both Paragraphs.
